### PR TITLE
Replace DEV with the community name... almost everywhere

### PR DESCRIPTION
**Pull request by Ridhwana**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Replace DEV with the community name... almost everywhere. 

I'll do some more find and replaces in a subsequent PR:
- I missed a few where I need to still make sure that nothing breaks. 
- I'll focus on mailers in a separate PR
- and more.... (baby steps for now)

## Related Tickets & Documents
https://github.com/thepracticaldev/dev.to/issues/6858

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

alt_text

### CODE REVIEW

File: app/views/pages/badges.html.erb [hunk 1]
```diff
@@ -49,12 +49,12 @@
   &lt;img src="https://d2fltix0v2e0sb.cloudfront.net/dev-badge.svg" alt="<%= current_user.name %>'s DEV Profile" height="30" width="30"&gt;
```
Comment (juliannatetreault) [inline app/views/pages/badges.html.erb:49]: Should the DEV here be replaced with `<%= community_name %>` as well? 👀

Comment (Ridhwana) [reply app/views/pages/badges.html.erb]: 🦅 Eagle eyes 😁 Thank you - yes it should! I pushed the update. 🎉

File: app/views/comments/_login_cta_comment.html.erb [hunk 2]
```diff
@@ -20,7 +20,7 @@
         </button>
       </div>
       <div class="actions">
-        <a href="/enter" id="cta-comment-register-now-link" class="register-now-cta" rel="nofollow">JOIN THE DEV COMMUNITY</a>
+        <a href="/enter" id="cta-comment-register-now-link" class="register-now-cta" rel="nofollow">JOIN THE <%= community_qualified_name %></a>
```
Comment (lightalloy) [inline app/views/comments/_login_cta_comment.html.erb:23]: Should we `upcase` here?

Comment (Ridhwana) [reply app/views/comments/_login_cta_comment.html.erb:23]: Nice catch! Yes.

Comment (Ridhwana) [reply app/views/comments/_login_cta_comment.html.erb]: Done!

File: app/views/pages/privacy.html.erb [hunk 3]
```diff
@@ -95,20 +95,20 @@
...
+      <%= community_name %> uses a variety of third-party services to diagnose errors and improve the performance of our site. We aim to minimize the amount of personal information shared, but the information may include your IP address or other identifying information. All service providers and third-party vendors are required to meet our data protection standards.
     </p>
     <p><strong>Payment processing</strong>
       <br>
-      DEV does not process payments directly — we rely on third-party services such as Stripe, Shopify, and Paypal to receive payments and store any payment information.
-      <a href="https://shop.dev.to">The DEV Shop</a> is run through Shopify, a third-party e-commerce platform with their own Privacy Policy.
+      <%= community_name %> does not process payments directly — we rely on third-party services such as Stripe, Shopify, and Paypal to receive payments and store any payment information.
+      <a href="https://shop.dev.to">The <%= community_name %> Shop</a> is run through Shopify, a third-party e-commerce platform with their own Privacy Policy.
```
[the remark this answers is not in the document]
Comment (Ridhwana) [reply app/views/pages/privacy.html.erb:107]: Yeah definitely there's a couple of support and dev specific emails that I've been making note of for generalisation that I'll add the the board to tackle in subsequent PR's. 
Following on that, I haven't also made any of the hrefs or urls dynamic based on the env variables (except in the meta tags). That will also be a separate PR.

File: app/views/partnerships/_bronze_sponsor_copy.html.erb [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <h1>Bronze Sponsorship</h1>
 
 <p>
-  Thank you for supporting the DEV Community!  Here’s what to expect with a Bronze Sponsorship:
+  Thank you for supporting the <%= community_name %> Community!  Here’s what to expect with a Bronze Sponsorship:
```
[the remark this answers is not in the document]
Comment (Ridhwana) [reply app/views/partnerships/_bronze_sponsor_copy.html.erb:4]: I will add this in my next PR, I blame my `find and replace` blindness 😁. I am going to give the codebase a thorough sweep next week again as I know I've missed quite a few.

**Pull request merged**